src/Morphir/Visual/ValueEditor.elm

```diff
@@ -272,6 +272,9 @@ initComponentState ir valueType maybeInitialValue =
                                     Type.CustomTypeSpecification _ constructors ->
                                         initCustomEditor ir fQName constructors maybeInitialValue
 
+                                    Type.DerivedTypeSpecification _ config ->
+                                        initComponentState ir config.baseType maybeInitialValue
+
```

Comment:
Great choice. Using the base type editor makes things work even if we don't have a specific editor.